modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -2139,6 +2147,39 @@ public void testDeleteObjectEntry() throws Exception {
 			_objectDefinition.getPluralLabelMap(), _objectDefinition.getScope(),
 			_objectDefinition.getStatus());
 
+		_objectEntryLocalService.deleteObjectEntry(objectEntry3);
+
+		_assertCount(0);
+
+		// Delete object entry with an object definition that is related to a
+		// draft object definition
+
+		_objectDefinition = _publishCustomObjectDefinition(
+			false,
+			Arrays.asList(
+				ObjectFieldUtil.createObjectField(
+					ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+					ObjectFieldConstants.DB_TYPE_STRING,
+					RandomTestUtil.randomString(), StringUtil.randomId())));
```

Comment:
Hey @thallescezar this variable is being used to delete another object definition that was created on setUp, once you create another object definition and change the variables to yours it'll delete the last one and will keep the one created on set up in the database, but actually I believe both object definitions are being deleted after test run, if so, please define the draft object definition and this new one here only in this scope.

Thank you!